### PR TITLE
Test Article - String to char arrray and vice versa

### PR DESCRIPTION
**Pull request by dsharew**
I have a doubt about my code formatter, I might need help.

### CODE REVIEW

File: core-java/src/test/java/com/baeldung/utils/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+        @Test public void givenString_whenCallingStringToCharArray_shouldConvertToCharArray() {
+                String givenString = "abcd";
+
+                char[] result = Utils.stringToCharArray(givenString);
+
+                assertTrue(Arrays.equals(result, new char[] { 'a', 'b', 'c', 'd' }));
+        }
+
+        @Test public void givenCharArray_whenCallingCharArrayToString_shouldConvertToString() {
```
Comment (pivovarit) [inline core-java/src/test/java/com/baeldung/utils/UtilsTest.java]: Did you use the default formatter?  The annotation in the same line as the method is a convention that we do not follow

File: core-java/src/test/java/com/baeldung/utils/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class UtilsTest {
```
Comment (pivovarit) [inline core-java/src/test/java/com/baeldung/utils/UtilsTest.java]: UtilsTest is a little bit too generic name. I can imagine multiple classes with similar names after having published hundreds of articles :)

File: core-java/src/main/java/com/baeldung/utils/Utils.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package com.baeldung.utils;
+
+import java.util.Arrays;
+
+public class Utils {
```
Comment (pivovarit) [inline core-java/src/main/java/com/baeldung/utils/Utils.java]: UtilsTest is a little bit too generic name. I can imagine multiple classes with similar names after having published hundreds of articles :)

File: core-java/src/main/java/com/baeldung/algorithms/annealing/City.java [hunk 4]
```diff
@@ -2,21 +2,35 @@
 
 import lombok.Data;
 
-@Data
-public class City {
+@Data public class City {
```
Comment (pivovarit) [inline core-java/src/main/java/com/baeldung/algorithms/annealing/City.java]: One note, is this class relevant to the scope of the article? Also, the formatting is a little bit off. I already addressed this in another comment

File: core-java/src/main/java/com/baeldung/utils/Utils.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
+package com.baeldung.utils;
```
Comment (pivovarit) [inline core-java/src/main/java/com/baeldung/utils/Utils.java]: Let's put the code in the matching package structure, in this case, we could use "com.baeldung.chartoarray"

File: core-java/src/main/java/com/baeldung/utils/Utils.java [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+        public static char[] stringToCharArray(final String input) {
+                if (null == input)
+                        return null;
+
+                return input.toCharArray();
+        }
+
+        public static String charArrayToString(final char[] charArray) {
+                if (null == charArray)
```
Comment (pivovarit) [inline core-java/src/main/java/com/baeldung/utils/Utils.java]: The formatting here does not follow the convention. Keep in mind that you have a predefined formatters for Eclipse and IDEA in the repo

**Comment by pivovarit** [comment]
@dsharew  Which IDE are you using?

**Comment by dsharew** [comment]
Hi,
I am using intellij 15.01.
Your comments they all make sense btw. I will fix them tomorrow in the
morning except I might need help with the intellij formatting.

Thanks,

File: core-java/src/test/java/com/baeldung/utils/UtilsTest.java (hunk 2 above)
Comment (dsharew) [inline core-java/src/test/java/com/baeldung/utils/UtilsTest.java]: I agree

File: core-java/src/test/java/com/baeldung/utils/UtilsTest.java (hunk 1 above)
Comment (dsharew) [inline core-java/src/test/java/com/baeldung/utils/UtilsTest.java]: Am having issue with my IDE will contact you tomorrow morning.

File: core-java/src/main/java/com/baeldung/algorithms/annealing/City.java (hunk 4 above)
Comment (dsharew) [inline core-java/src/main/java/com/baeldung/algorithms/annealing/City.java]: No but this class had an issue was not able to build the project without the minor changes i added

Comment (pivovarit) [inline core-java/src/main/java/com/baeldung/algorithms/annealing/City.java]: Lack of getters was not a problem here. The @Data annotation that is present here is taking care of this if you enable annotation processing in your IDE

**Comment by pivovarit** [comment]
@dsharew Any updates here?
